docs/site/Creating-components.md

```diff
@@ -171,13 +170,7 @@ The idiomatic solution has two parts:
         @inject('sequence.actions.reject') reject: Reject,
         // Inject the new action here:
         @inject('authentication.actions.authenticate') authenticate
-      ) {
-        this.findRoute = findRoute;
```

Comment:
You need to add `private`, `protected`, or `private` before each argument to tell TypeScript to create a new property for each argument.

```ts
@inject('sequence.actions.findRoute') private findRoute,
```

Please consider adding type information to all arguments where it's missing now.